sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -51,6 +51,14 @@ namespace Azure { namespace Core {
      */
     virtual std::string GetResumeToken() const = 0;
 
+    /**
+     * @brief Get the raw HTTP response.
+     * @return A pointer to #Azure::Core::Http::RawResponse or null if the RawResponse does not
+     * exist.
```

Comment:
Since we haven't designed for this yet (and because I don't think we should return null anyway), let's not mention that in the docs **yet**.

```suggestion
     * @return A pointer to #Azure::Core::Http::RawResponse.
```